pages/book.vue

```diff
@@ -0,0 +1,215 @@
+<script setup lang="ts">
+definePageMeta({
+  layout: false,
+});
+
+const booklist = [
+  {
+    id: 1,
+    image: "~/assets/images/elon.jpeg",
+    name: "Elon Mask",
+    description: "Ashlee Vance",
+    icon: 3,
+  },
+  {
+    id: 2,
+    image: "~/assets/images/tools.jpeg",
+    name: "Tools of Titans",
+    description: "Team Feriss",
+    icon: 3,
+  },
+  {
+    id: 3,
+    image: "~/assets/images/jen.jpeg",
+    name: "You are a badass",
+    description: "Jen Sincero",
+    icon: 2,
+  },
+];
+</script>
+
+<template>
+  <div class="wrapper">
+    <div class="top-nav">
+      <i class="bi bi-list" style="font-size: 2rem"></i>
+      <h2>Bookstop</h2>
+    </div>
+    <div class="searchbar">
+      <form class="d-flex" role="search">
+        <input class="form-control me-2" type="search" placeholder="Search" aria-label="Search" />
+        <button class="btn btn-outline-primary" type="submit">Search</button>
+      </form>
+    </div>
+    <div class="pc-tab">
+      <input id="tab1" checked type="radio" name="pct" />
```

Comment:
`name="pct"` declaration doesn't mean anything. Give a meaningful name so that developer can understand.